code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -35,35 +35,41 @@ client/proc/one_click_antag()
 	var/list/mob/living/carbon/human/candidates = list()
 	var/mob/living/carbon/human/H = null
 
-	log_admin("[key_name(owner)] tried making Traitors with One-Click-Antag")
-	message_admins("[key_name_admin(owner)] tried making Traitors with One-Click-Antag")
-
-	for(var/mob/living/carbon/human/applicant in player_list)
-		if(ROLE_TRAITOR in applicant.client.prefs.be_special)
-			if(player_old_enough_antag(applicant.client,ROLE_TRAITOR))
-				if(!applicant.stat)
-					if(applicant.mind)
-						if(!applicant.mind.special_role)
-							if(!jobban_isbanned(applicant, "traitor") && !jobban_isbanned(applicant, "Syndicate"))
-								if(!(applicant.mind.assigned_role in temp.restricted_jobs))
-									if(!(applicant.client.prefs.species in temp.protected_species))
-										candidates += applicant
-
-	if(candidates.len)
-		var/numTratiors = min(candidates.len, 3)
-
-		for(var/i = 0, i<numTratiors, i++)
-			H = pick(candidates)
-			H.mind.make_Tratior()
-			candidates.Remove(H)
-
-		return 1
-
-
+	var/antnum = input(owner, "How many traitors you want to create? (0 - 5)","Amount:", 0) as num
+	var/confirmed = alert(owner, "Are you sure?", "Confirm creation", "Yes", "No")
+
+	if(!antnum)
+		return
+	if(antnum > 5 || antnum <= 0)
+		return
+	if(confirmed == "Yes")
+		log_admin("[key_name(owner)] tried making [antnum] Traitors with One-Click-Antag")
+		message_admins("[key_name_admin(owner)] tried making [antnum] Traitors with One-Click-Antag")
+
+		for(var/mob/living/carbon/human/applicant in player_list)
+			if(ROLE_TRAITOR in applicant.client.prefs.be_special)
+				if(player_old_enough_antag(applicant.client,ROLE_TRAITOR))
+					if(!applicant.stat)
+						if(applicant.mind)
+							if(!applicant.mind.special_role)
+								if(!jobban_isbanned(applicant, "traitor") && !jobban_isbanned(applicant, "Syndicate"))
+									if(!(applicant.mind.assigned_role in temp.restricted_jobs))
+										if(!(applicant.client.prefs.species in temp.protected_species))
+											candidates += applicant
+
+		if(candidates.len)
+			var/numTraitors = min(candidates.len, antnum)
+
+			for(var/i = 0, i<numTraitors, i++)
+				H = pick(candidates)
+				H.mind.make_Traitor()
+				candidates.Remove(H)
+
+			return 1
 	return 0
 
 
-/datum/admins/proc/makeChanglings()
+/datum/admins/proc/makeChangelings()

```

Comment:
`makeChangelings` and `make_Traitors()` are not very consistent. You probably want to stick to the first style.

Reply:
Notice the plurals. The procs are `makeTraitors()` and `makeChangelings()` or make_Traitor() / `make_Changeling()`.

Reply:
I'd like to stick with the original differentiation

Reply:
But why? It makes no sense to do so.

Reply:
One is a proc for the mind and the other is for the process of making one click antags. The differentation help make sure you don't confuse the two.

Reply:
Fine by me.

Reply:
Fuck all of gamemode and mind code's formatting. Don't worry about this.